packages/medusa/src/api-v2/admin/fulfillments/validators.ts

```diff
@@ -3,5 +3,42 @@ import { createSelectParams } from "../../utils/validators"
 
 export const AdminFulfillmentParams = createSelectParams()
 
-export const AdminCancelFulfillment = z.object({})
 export type AdminCancelFulfillmentType = z.infer<typeof AdminCancelFulfillment>
+export const AdminCancelFulfillment = z.object({})
+
+export type AdminCreateFulfillmentType = z.infer<typeof AdminCreateFulfillment>
+export const AdminCreateFulfillment = z.object({
+  location_id: z.string(),
+  provider_id: z.string(),
+  delivery_address: z.object({
```

Comment:
There is an `AddressPayload` in `common-validators` (it was used in the orders), should we standardize the model and use that instead?

Reply:
nice! I've addressed this [here](https://github.com/medusajs/medusa/pull/7118/files#diff-3717666eaae8ef3178a183f8f85232d20d40e56d75d1edf6f534859c6147d96e)